src/cljs/harja/views/hallinta.cljs

```diff
@@ -49,6 +50,12 @@
      ^{:key "integraatioloki"}
      [integraatioloki/integraatioloki])
 
+   "Yhteydenpito"
+   :yhteydenpito
+   (when (oikeudet/hallinta)
```

Comment:
Näyttäis että kaikille näkymille on tähän mennessä tehty oma excel-rivi. Varmaan siis kannattaisi lisätä se sinne.